test/functional/mempool_accept_v3.py

```diff
@@ -262,6 +262,16 @@ def test_v3_ancestors_package(self):
         result = node.testmempoolaccept([tx_v3_parent["hex"], tx_v3_child["hex"], tx_v3_grandchild["hex"]])
         assert all([txresult["package-error"] == f"v3-violation, tx {tx_v3_grandchild['txid']} (wtxid={tx_v3_grandchild['wtxid']}) would have too many ancestors" for txresult in result])
 
+        # Check that if parent is in mempool and child->grandchild package(as a single chunk) is submitted, this also fails limit checks
+        tx_v3_child = self.wallet.create_self_transfer(utxo_to_spend=tx_v3_parent["new_utxo"], fee_rate = 0, version=3)
+        tx_v3_grandchild = self.wallet.create_self_transfer(utxo_to_spend=tx_v3_child["new_utxo"], version=3)
+
+        node.sendrawtransaction(tx_v3_parent["hex"])
+        self.check_mempool([tx_v3_parent["txid"]])
+
+        result = node.submitpackage([tx_v3_child["hex"], tx_v3_grandchild["hex"]])
+        assert_equal(result["package_msg"], f"v3-violation, tx {tx_v3_grandchild['txid']} (wtxid={tx_v3_grandchild['wtxid']}) would have too many ancestors")
+
     @cleanup(extra_args=["-acceptnonstdtxn=1"])
     def test_v3_ancestors_package_and_mempool(self):
```

Comment:
Is this scenario (parent is in mempool, child->grandchild package is submitted and is rejected) at least in part already covered in the `test_v3_ancestors_package_and_mempool()` test?  Maybe I'm overlooking something?

```python
    def test_v3_ancestors_package_and_mempool(self):
        """
        A v3 transaction in a package cannot have 2 v3 parents.
        Test that if we have a transaction graph A -> B -> C, where A, B, C are
        all v3 transactions, that we cannot use submitpackage to get the
        transactions all into the mempool.

        Verify, in particular, that if A is already in the mempool, then
        submitpackage(B, C) will fail.
        """
```

Reply:
ya know what, it is. Closing this PR.